aleph_alpha_client/completion.py

```diff
@@ -0,0 +1,116 @@
+
+        frequency_penalty (float, optional, default 0.0)
+            The frequency penalty reduces the likelihood of generating tokens that are already present in the text. Presence penalty is dependent on the number of occurences of a token.
+
+        repetition_penalties_include_prompt (bool, optional, default False)
+            Flag deciding whether presence penalty or frequency penalty are applied to the prompt and completion (True) or only the completion (False)
+
+        use_multiplicative_presence_penalty (bool, optional, default True)
+            Flag deciding whether presence penalty is applied multiplicatively (True) or additively (False). This changes the formula stated for presence and frequency penalty.
+
+        best_of (int, optional, default None)
+            Generates best_of completions server-side and returns the "best" (the one with the highest log probability per token). Results cannot be streamed.
+            When used with n, best_of controls the number of candidate completions and n specifies how many to return – best_of must be greater than n.
+
+        n (int, optional, default 1)
+            How many completions to generate for each prompt.
+
+        logit_bias (dict mapping token ids to score, optional, default None)
+            The logit bias allows to influence the likelihood of generating tokens. A dictionary mapping token ids (int) to a bias (float) can be provided. Such bias is added to the logits as generated by the model.
+
+        log_probs (int, optional, default None)
+            Number of top log probabilities to be returned for each generated token. Log probabilities may be used in downstream tasks or to assess the model's certainty when producing tokens.
+
+        stop_sequences (List(str), optional, default None)
+            List of strings which will stop generation if they're generated. Stop sequences may be helpful in structured texts.
+
+            Example: In a question answering scenario a text may consist of lines starting with either "Question: " or "Answer: " (alternating). After producing an answer, the model will be likely to generate "Question: ". "Question: " may therfore be used as stop sequence in order not to have the model generate more questions but rather restrict text generation to the answers.
+
+        tokens (bool, optional, default False)
+            return tokens of completion
+
+        disable_optimizations  (bool, optional, default False)
+            We continually research optimal ways to work with our models. By default, we apply these optimizations to both your prompt and  completion for you.
+
+            Our goal is to improve your results while using our API. But you can always pass disable_optimizations: true and we will leave your prompt and completion untouched.
+    """
+
+    prompt: Union[str, List[Union[str, ImagePrompt]]] = ""
+    maximum_tokens: Optional[int] = 64
```

Comment:
Does it make sense to have those parameters with not-`None` default `Optional`?

Reply:
nope